### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by ngAtesz**
Read my comments below.

### CODE REVIEW

File: src/Codecool.FilePartReader.UnitTests/FilePartReaderTests.cs [hunk 1]
```diff
@@ -19,6 +21,7 @@ public class FilePartReaderTest
         public void Setup()
         {
             _filePartReader = new FilePartReader();
+            // comment?
             //_filePartReader.Setup();
```
Comment (ngAtesz) [inline src/Codecool.FilePartReader.UnitTests/FilePartReaderTests.cs:25]: Do not push commented code into remote repository.

File: src/Codecool.FilePartReader.UnitTests/FilePartReaderTests.cs [hunk 2]
```diff
@@ -64,6 +67,7 @@ public void Setup_FromLineBiggerThanToLine_HasNoReturn()
         [Test]
         public void Setup_FromLineIsSmallerThanOne_HasNoReturn()
         {
+            // Arrange? Act?
```
Comment (ngAtesz) [inline src/Codecool.FilePartReader.UnitTests/FilePartReaderTests.cs:70]: Split into these certain steps not just write oneliners.

Comment (ngAtesz) [reply src/Codecool.FilePartReader.UnitTests/FilePartReaderTests.cs:70]: The naming is misleading and it's eerly similar to Rajmunds method naming. Actually in the name you say `HasNoReturn` but in the method you expect an `ArgumentException`. Rename your method accordingly

Comment (ngAtesz) [reply src/Codecool.FilePartReader.UnitTests/FilePartReaderTests.cs:70]: What about that case when the `toLine` parameter is less than 0 or more than the length of the file? What are the expected behavior in those cases?

File: src/Codecool.FilePartReader.UnitTests/FilePartReaderTests.cs [hunk 3]
```diff
@@ -79,6 +83,7 @@ public void ReadLines_CreateString_ReturnResult()
         [Test]
         public void GetWordsOrderedAlphabetically_SortWords_ReturnSortedList()
```
Comment (ngAtesz) [inline src/Codecool.FilePartReader.UnitTests/FilePartReaderTests.cs:84]: This is a `FileWorldAnalyzer` test not `FilePartReaderTest`. It should be in a separated file.

File: src/Codecool.FilePartReader.UnitTests/FilePartReaderTests.cs [hunk 4]
```diff
@@ -79,6 +83,7 @@ public void ReadLines_CreateString_ReturnResult()
...
         {
+            // This is an Analyzer test not FilePartReader test -> should be in separated test file
             _filePartReader.Setup("test.txt", 2, 6);
```
Comment (ngAtesz) [inline src/Codecool.FilePartReader.UnitTests/FilePartReaderTests.cs:87]: `text.txt` arise multiple places. In this case extract it into a private field or into the test `Setup` method.

**Review by ngAtesz — COMMENTED** [review]
There are some missing test cases and some of the test methods are one liners which is not recommended:

- `Setup()` got a non-existing file name.
- Analyzer -> `ReadLines(1,1)`

**Review by gergomahler — APPROVED** [review]

**Pull request merged**